nltk/metrics/distance.py

```diff
@@ -66,7 +68,7 @@ def edit_distance(s1, s2, substitution_cost=1, transpositions=False):
     been done in other orders, but at least three steps are needed.
 
     Allows specifying the cost of substitution edits (e.g., "a" -> "b"),
-    because sometimes it makes sense to assign greater penalties to substitutions.
+    because often it makes sense to assign bigger penalties to substitutions.
```

Comment:
**Is this the right assumption that it's often and not sometimes?** 

Also `greater penalties` sounds better than `bigger penalties`, the latter sounds presidential ;P